### PR TITLE
Migrate Project langding page.

### PR DESCRIPTION
**Pull request by wingtofree**
Finish [DevOps Task 231652](https://dev.azure.com/ceapex/Engineering/_workitems/edit/231652)
Following this guide: [Create or update a landing page](https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-landing-page?branch=master)

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[c5accc8](https://github.com/wingtofree/office-js-docs-pr/commits/c5accc83efad26ebe0fe091fff6175693f7569aa)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/toc.yml](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-migrate-project/docs/toc.yml) | :warning:Warning | | [Details](#user-content-a2f50e64-bb60066e)
[docs/index.md](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-migrate-project/docs/index.md) | :warning:Warning | | [Details](#user-content-d93b521a-bb60066e)
[docs/reference/overview/project-add-ins-reference-overview.md](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-migrate-project/docs/reference/overview/project-add-ins-reference-overview.md) | :warning:Warning | | [Details](#user-content-c7181a99-bb60066e)
[docs/project/index.md](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-migrate-project/docs/project/index.md) | :white_check_mark:Succeeded | |
[docs/project/index.yml](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-migrate-project/docs/project/index.yml) | :white_check_mark:Succeeded | |

<a id="a2f50e64-bb60066e"></a>
### [docs/toc.yml](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-migrate-project/docs/toc.yml)
 - **Line 0**: **[Warning-[InvalidFileLink](https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-links?branch=master)]** `````Invalid file link:(~/project/index.md).`````

<a id="d93b521a-bb60066e"></a>
### [docs/index.md](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-migrate-project/docs/index.md)
 - **Line 0**: **[Warning-[InvalidFileLink](https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-links?branch=master)]** `````Invalid file link:(~/project/index.md).`````

<a id="c7181a99-bb60066e"></a>
### [docs/reference/overview/project-add-ins-reference-overview.md](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-migrate-project/docs/reference/overview/project-add-ins-reference-overview.md)
 - **Line 19**: **[Warning-[InvalidFileLink](https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-links?branch=master)]** `````Invalid file link:(~/project/index.md).`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C6%5C8%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202006080819152897-1932%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=PzMuaeriTk%2BRI5N60KwlPukM0KeGLO6GRlugiEi7Jj8%3D&st=2020-06-08T08%3A15%3A31Z&se=2020-07-09T08%3A20%3A31Z&sp=r).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://teams.microsoft.com/l/message/19:7ecffca1166a4a3986fed528cf0870ee@thread.skype/1590030602688?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&parentMessageId=1590030602688&teamName=Docs%20Support&channelName=General&createdTime=1590030602688) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by ElizabethSamuel-MSFT** [comment]
Closing and reopening to generate preview site.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[c5accc8](https://github.com/wingtofree/office-js-docs-pr/commits/c5accc83efad26ebe0fe091fff6175693f7569aa)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/toc.yml](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-migrate-project/docs/toc.yml) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/project/index?branch=pr-en-us-1932) | [Details](#user-content-a2f50e64-a03e490d)
[docs/index.md](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-migrate-project/docs/index.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/index?branch=pr-en-us-1932) | [Details](#user-content-d93b521a-a03e490d)
[docs/reference/overview/project-add-ins-reference-overview.md](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-migrate-project/docs/reference/overview/project-add-ins-reference-overview.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/overview/project-add-ins-reference-overview?branch=pr-en-us-1932) | [Details](#user-content-c7181a99-a03e490d)
[docs/project/index.md](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-migrate-project/docs/project/index.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/overview/project-add-ins-reference-overview?branch=pr-en-us-1932) |
[docs/project/index.yml](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-migrate-project/docs/project/index.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/project/index?branch=pr-en-us-1932) |

<a id="a2f50e64-a03e490d"></a>
### [docs/toc.yml](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-migrate-project/docs/toc.yml)
 - **Line 0**: **[Warning-[InvalidFileLink](https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-links?branch=master)]** `````Invalid file link:(~/project/index.md).`````

<a id="d93b521a-a03e490d"></a>
### [docs/index.md](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-migrate-project/docs/index.md)
 - **Line 0**: **[Warning-[InvalidFileLink](https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-links?branch=master)]** `````Invalid file link:(~/project/index.md).`````

<a id="c7181a99-a03e490d"></a>
### [docs/reference/overview/project-add-ins-reference-overview.md](https://github.com/wingtofree/office-js-docs-pr/blob/qinglin-migrate-project/docs/reference/overview/project-add-ins-reference-overview.md)
 - **Line 19**: **[Warning-[InvalidFileLink](https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-links?branch=master)]** `````Invalid file link:(~/project/index.md).`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C6%5C8%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202006081628214791-1932%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=AIukHfx%2FfXafw%2FdzWYnyDM4GxZXcqWneG8MFkrJZnQs%3D&st=2020-06-08T16%3A24%3A53Z&se=2020-07-09T16%3A29%3A53Z&sp=r).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://teams.microsoft.com/l/message/19:7ecffca1166a4a3986fed528cf0870ee@thread.skype/1590030602688?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&parentMessageId=1590030602688&teamName=Docs%20Support&channelName=General&createdTime=1590030602688) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/project/index.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+title: Project Add-ins documentation # < 60 chars
+summary: With Project add-ins, you can use familiar web technologies such as HTML, CSS, and JavaScript to build a solution that can run in Project on Windows.  # < 160 chars
+
+metadata:
+  title: Project Add-ins documentation # Required; page title displayed in search results. Include the brand. < 60 chars.
+  description: With Project add-ins, you can use familiar web technologies such as HTML, CSS, and JavaScript to build a solution that can run in Project on Windows.  # Required; article description that is displayed in search results. < 160 chars.
+  ms.service: o365-proplus-itpro #Required; service per approved list. service slug assigned to your service by ACOM.
+  ms.topic: landing-page # Required
+  ms.date: 07/05/2019 #Required; mm/dd/yyyy format.
```
Comment (Rick-Kirkham) [inline docs/project/index.yml:11]:
```suggestion
  ms.date: 06/08/2020 #Required; mm/dd/yyyy format.
```

**Pull request merged**